.github/workflows/test-ecosystem.yml

```diff
@@ -0,0 +1,131 @@
+name: Test Ecosystem
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *' # Every day at 00:00 UTC
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        # Based on https://www.npmjs.com/search?q=keywords%3Astylelint-plugin&ranking=popularity
+        # The following snippet on a browser dev console is helpful to create this list.
+        #
+        #     [...document.querySelectorAll('a[href^="/package"][target="_self"]')].map(a => a.href.replace(/.+\/package\/(.+)$/, "- '$1'")).sort().join("\n")
+        #
+        package:
```

Comment:
This reminds me of https://github.com/stylelint/stylelint/issues/7396#issuecomment-1859205249

Reply:
Thanks for the feedback. It sounds great to me. 👍🏼 

I want to know a way to create a more effective package list. <https://www.npmjs.com/search> seems insufficient for this purpose. 😓